src/qml/editorwidgets/TextEdit.qml

```diff
@@ -8,6 +8,8 @@ import "."
 EditorWidgetBase {
   id: topItem
 
+  property bool isEditable: isEnabled && typeof(value) != object
```

Comment:
```suggestion
  property bool isEditable: isEnabled && value && typeof(value) !== 'object'
```
note `typeof(null) === 'object'`

Have you tried this, the `object` part should be in quotes, no?